utils/directories.cc

```diff
@@ -131,7 +141,7 @@ void directories::override_if_empty(std::vector<fs::path>& v, const fs::path& de
     }
 }
 
-future<> directories::create_and_verify(directories::set dir_set) {
+future<> directories::create_and_verify(directories::set&& dir_set) {
```

Comment:
Why is this change in this patch? If it's copy optimization it can be done in its own patch

Reply:
Fixed. I used a reference without much thinking - directories::set is only forward declared in the header file.

However, it is a private member function and it is used only in the source file. Without the reference it compiles. I reverted that change.